src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/util/AssignmentsFactory.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.util
+
+import org.apache.commons.lang3.RandomStringUtils.randomAlphanumeric
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.AuthUser
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ReferralAssignment
+import java.time.OffsetDateTime
+
+class AssignmentsFactory(em: TestEntityManager? = null) : EntityFactory(em) {
+  private val authUserFactory = AuthUserFactory(em)
+
+  fun create(numberOfAssignments: Int): List<ReferralAssignment> {
+    val assignedUsers = mutableListOf<AuthUser>().apply {
+      repeat(numberOfAssignments) {
+        this.add(authUserFactory.create(randomAlphanumeric(6), randomAlphanumeric(5), randomAlphanumeric(12)))
+      }
+    }
+    return createInOrder(*assignedUsers.toTypedArray())
+  }
+
+  fun createInOrder(vararg users: AuthUser): List<ReferralAssignment> {
```

Comment:
Do we need to name it as createInOrder as we are not maintaining any order and things is the new one superseds the old one

Reply:
@sldblog , It was just a clarification. Looking good